bin/service-url-include.sh

```diff
@@ -23,10 +26,10 @@ function get_k8s_info {
   object=$2
   info_item=$3
 
-  info=$(kubectl -n $namespace get $object  -o=jsonpath=${json_paths[$info_item]})
+  info=$(kubectl -n $namespace get $object  -o=jsonpath=${json_paths[$info_item]} 2>/dev/null)
   rc=$?
 
-  if [ "$rc" == "0" ]; then
+  if [ ! -z  "$info" ]; then
```

Comment:
minor nit that you don't even have to fix, but you can use `-n` in place of `! -z`